src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -117,7 +117,8 @@ impl RuntimeHandlerManagerInner {
         let shim_mgmt_svr = MgmtServer::new(
             &self.id,
             self.runtime_instance.as_ref().unwrap().sandbox.clone(),
-        );
+        )
+        .context("Failed to spawn shim_mgmt_svr")?;
```

Comment:
Nit: consistency: there are a few references to "shim_mgmt" in errors, but also (the much clearer imho) `"Failed to create shim-mgmt server"`. I'd consider making all the error messages consistent and spelling the message out in full by using a `const` to avoid duplication and ensure consistency. Something like:

```
const ERR_NO_SHIM_SERVER: &str = "Failed to create shim management server";
```

Reply:
I agree, but I also find out a problem of kata right now.

We currently are abusing the `anyhow`, instead of implementing our own `Error` types. Abusing crates like `anyhow` and `thiserror` 's formatting tricks are not idiomatic IMO.

WDYT @jodh-intel

Reply:
@Ji-Xinyou - yes, I tend to agree. Could you raise an issue so this topic can be discussed separately?

Reply:
@jodh-intel I have raised an issue. https://github.com/kata-containers/kata-containers/issues/5918

Reply:
Thanks @Ji-Xinyou ! :smile: